### PR TITLE
Bugfix

### PR DESCRIPTION
**Pull request by ElonPark**
🐛 JSON object has no attribute 'next_max_id'
🐛 JSON object has no attribute 'items'
🐛 'bool' object is not subscriptable

### CODE REVIEW

File: instabot/bot/bot_unfollow.py [hunk 1]
```diff
@@ -4,22 +4,25 @@
 def unfollow(self, user_id):
     user_id = self.convert_to_user_id(user_id)
     user_info = self.get_user_info(user_id)
-    username = user_info["username"]
+    username = user_info.get("username", "")
```
Comment (basnijholt) [inline instabot/bot/bot_unfollow.py]: Shouldn't the username always be there?

**Review by basnijholt — COMMENTED** [review]
Seems legit! Just have one question.

**Comment by basnijholt** [comment]
And could you fix the trailing whitespace "error"?

**Comment by ElonPark** [comment]
I think there is no problem. But if you think so, I think there is a way to use try ~ except in the whole function.

**Review by sudoguy — APPROVED** [review]

**Pull request merged**